lib/common/pdf_helpers.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'hexapdf'
+require './lib/sentry_logging'
+
+module Common
+  module PdfHelpers
+    extend SentryLogging
+
+    def self.unlock_pdf(input_file, password, output_file)
+      doc = HexaPDF::Document.open(input_file, decryption_opts: { password: })
+      doc.encrypt(name: nil)
+      doc.write(output_file)
+    rescue HexaPDF::EncryptionError => e
+      log_message_to_sentry(e.message, 'warn')
```

Comment:
I wonder if we should also add a `Rails.logger.<info || error || etc>` so we can monitor these issues from DataDog too? 🤔